pg_search/src/postgres/options.rs

```diff
@@ -117,7 +138,7 @@ fn cstr_to_rust_str(value: *const std::os::raw::c_char) -> String {
 }
 
 // For now, we support changing the tokenizer between default, raw, and en_stem
```

Comment:
This comment is outdated with the new stemmers, no?

Reply:
Yeah I'll just remove it, not really the place for API documentation.